### PR TITLE
slim app banner

### LINKED ISSUE #250: new app banner variant
image

### PR DESCRIPTION
**Pull request by ravverma**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #250

Test URLs:
- Before: https://main--icicidirect--aemsites.hlx.live/draft/ravverma/app-download
- After: https://slim-app-banner--icicidirect--aemsites.hlx.live/draft/ravverma/app-download

### CODE REVIEW

**Comment by ravverma** [comment]
image

File: blocks/app-download/app-download.css [hunk 1]
```diff
@@ -58,6 +62,15 @@
   display: block;
 }
 
+.block.app-download .store a img {
+  width: 183px;
+  height: 54px;
+}
+
+.block.app-download.slim .store a picture{
```
Comment (sdmcraft) [inline blocks/app-download/app-download.css:70]: Nit: Indenting looks messed up.

**Comment by sdmcraft** [comment]
Please add this variant in the block library.

File: blocks/app-download/app-download.css [hunk 2]
```diff
@@ -58,6 +62,15 @@
   display: block;
 }
 
+.block.app-download .store a img {
+  width: 183px;
```
Comment (sdmcraft) [inline blocks/app-download/app-download.css:66]: Do we need both dimensions to be specified? Specifying both of them could potentially distort the image if that does not match image's aspect ratio.

Comment (ravverma) [reply blocks/app-download/app-download.css:66]: not need, only width required.

File: blocks/app-download/app-download.css (hunk 1 above)
Comment (ravverma) [reply blocks/app-download/app-download.css:70]: done

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**